protos/flyteidl/event/event.proto

```diff
@@ -167,4 +185,10 @@ message TaskExecutionMetadata {
         INTERRUPTIBLE = 1;
     }
     InstanceClass instance_class = 16;
+
+    // Generated unique name for this task execution used by the backend.
+    string generated_name = 1;
+
+    // Includes information about how resource token allocation (if applicable).
+    ManagedResourceInfo managed_resource_info = 2;
```

Comment:
should this be repeated?

Reply:
perhaps i'm misunderstanding but isn't this only determined once per execution?

Reply:
A plugin can request multiple resource allocations during execution... our plugins only request one resource at most because we didn't need more... but the API allows plugins to request more

Reply:
@EngHabu got it, thanks. updated